solutions/standard/variables.tf

```diff
@@ -0,0 +1,660 @@
+##############################################################################
+# Common
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key used to provision resources."
+  sensitive   = true
+}
+
+variable "existing_secrets_manager_guid" {
```

Comment:
Please be consistent with all our other DAs. We always ask for CRN in DAs, not GUID